kvbc/include/kvbc_key_types.hpp

```diff
@@ -20,6 +20,13 @@ static const char reconfiguration_download_key = 0x26;
 static const char reconfiguration_install_key = 0x27;
 static const char reconfiguration_key_exchange = 0x28;
 static const char reconfiguration_add_remove = 0x29;
-static const char reconfiguration_client_key_exchange = 0x30;
+// 0x2a is reserved
```

Comment:
`static const char reconfiguration_reserved = 0x2a; //TODO [YB] check`

it shouldn't be reserved at a higher level